packages/tadviewer/src/components/GridPane.tsx

```diff
@@ -16,6 +16,7 @@ import log from "loglevel";
 
 import { ColumnType, NumericColumnHistogramData, Schema } from "reltab";
 import ReactDOM from "react-dom/client";
+import { CellClickData } from "./TadViewerPane";
```

Comment:
Better to move this type to a separate file to avoid the cyclic file dependencies (TadViewerPane -> AppPane -> GridPane -> TadViewerPane, and also TadViewerPane -> AppPane -> TadViewerPane).